docs/how-to/python_api.md

```diff
@@ -0,0 +1,188 @@
+## Use the Spark Client from Python 
+
+In the following, we show how to use the spark client to manage service accounts
+using Python.
+
+### Bind to Kubernetes
+
+The following snipped allows you to import relevant environment variables
+into a confined object, among which there should an auto-inference of your 
+kubeconfig file location. 
+
+```python
+import os
+from spark_client.domain import Defaults
+from spark_client.services import KubeInterface
+
+# Defaults for spark-client
+defaults = Defaults(dict(os.environ))  # General defaults
+
+# Create a interface connection to k8s
+kube_interface = KubeInterface(defaults.kube_config)
```

Comment:
This will change due to lightkube being introduced. Right now I have named the AbstractKubeInterface implementation as LightKube. Let's discuss naming in the other PR.

Reply:
This is fine. We can change this to LightKube after the PR gets merged (depending which one we merge first).